toast-tk-web-agent/src/main/java/io/toast/tk/agent/web/ScriptInjector.java

```diff
@@ -10,14 +10,15 @@
 import org.apache.commons.lang3.StringUtils;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.apache.commons.lang3.StringUtils'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)